content/en/tracing/trace_collection/tracing_naming_convention/_index.md

```diff
@@ -0,0 +1,128 @@
+| `component` | `string` | The name of the library/integration which created the span                                                                                                                                                                                                                        |
+
+### Network communications
+The following span tags can be used to describe work units corresponding to network communications:
+
+| **Fullname**                    | **Type** | **Description**                                                           |
+|---------------------------------|----------|---------------------------------------------------------------------------|
+| `network.client.ip`             | `string` | The IP address of the client that initiated the inbound connection        |
+| `network.destination.ip`        | `string` | The IP address where the outbound connection is being made to             |
+| `network.host.ip`               | `string` | Local host IP address                                                     |
+| `network.client.port`           | `number` | The port of the client that initiated the connection                      |
+| `network.destination.port`      | `number` | Remote port number of the outbound connection                             |
+| `network.client.name`           | `string` | The hostname of the client that initiated the inbound connection          |
+| `network.destination.name`      | `string` | Remote hostname or similar where the outbound connection is being made to |
+| `network.host.name`             | `string` | Local hostname                                                            |
+| `network.client.transport`      | `string` | Transport protocol used to make the inbound connection                    |
+| `network.destination.transport` | `string` | Transport protocol used to make the outbound connection                   |
+
+### HTTP requests
+The following span tags can be used to describe HTTP client and server spans:
+
+| **Fullname**                                | **Type** | **Description**                                                                                                                                                    |
+|---------------------------------------------|----------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `http.status_code`                          | `string` | The HTTP response status code                                                                                                                                      |
+| `http.url`                                  | `string` | The URL of the HTTP request, including the obfuscated query string. For more informations on obfuscation refer to [Data security][2] |
```

Comment:
we should document `http.url_details.*` tags as well.... once http url parsing is implemented on the backend